wallace/custom.py

```diff
@@ -215,27 +205,27 @@ def node():
                 }
```

Comment:
The nested try/except in here (line 209/199 in this diff) is another place that would be better served by catching a particular exception type.  Assuming that `exp.known_classes` is a dict-like object, then you should be anticipating a `KeyError`.  

You should internalize the rule of thumb that a *bare except* (using the except clause without specifying one or more exception type to be caught) is a bad idea.